manifests/satellite6.pp

```diff
@@ -0,0 +1,19 @@
+# Class: satellite6
+#   Install the SAtellite 6 flavor of the robottelo_slave
+#
+class robottelo_slave::satellite6 {
+  include ::robottelo_slave
+
+  ensure_packages([
+    'brewkoji',
+  ])
+
+  $rh_mirror='http://download.lab.bos.redhat.com'
```

Comment:
I don't think we can pull this from Jenkins, we'd need to configure this at application time or use a configuration file that we pull from downstream. The other option would be to move to using a foreman/Satellite to manage our slaves and setting parameters that way